client/js/components/forms/TeamForm.js

```diff
@@ -19,29 +20,36 @@ class TeamForm extends Component {
 
   render() {
     const { errors } = this.props;
-
-    let header = <h3>Add team</h3>;
-    if (this.state.id) {
-      header = <h3>Edit team #{this.state.id}</h3>;
-    }
-
     return (
-      <div>
+      <div id="TeamForm">
         <ErrorList errors={errors} showFieldErrors/>
-        {header}
-        <form id="teamEditForm" onSubmit={this.onSubmit.bind(this)}>
-          <div className="row">
-            <input type="text" placeholder="Name" value={this.state.name} minLength="2" maxLength="32" onChange={this.onInputChange.bind(this, 'name')} required/>
-          </div>
-          <div className="row">
-            <input type="text" placeholder="Summary" value={this.state.summary} maxLength="128" onChange={this.onInputChange.bind(this, 'summary')} required/>
-          </div>
-          <div className="row">
-            <textarea placeholder="Description" value={this.state.description} maxLength="1024" onChange={this.onInputChange.bind(this, 'description')}/>
-          </div>
-          <div className="row">
-            <button className="button" type="submit">Save</button>
-          </div>
+        <form onSubmit={this.onSubmit.bind(this)}>
+          <fieldset className="form-group">
+            <label>Team name*</label>
+            <input type="text" className="form-control" placeholder="Name" value={this.state.name}
+              minLength="2" maxLength="32" onChange={this.onInputChange.bind(this, 'name')} required/>
```

Comment:
Or let's do it differently - once PR is ready, I'll add another commit that would update client and server validation rules so that they are the same